src/covid_model_seiir_pipeline/forecasting/task/beta_forecast.py

```diff
@@ -23,21 +23,38 @@ def run_beta_forecast(draw_id: int, forecast_version: str, scenario_name: str, *
 
     logger.info('Loading input data.')
     location_ids = data_interface.load_location_ids()
+    # We'll use the same params in the ODE forecast as we did in the fit.
+    beta_params = data_interface.load_beta_params(draw_id=draw_id)
     # Thetas are a parameter generated from assumption or OOS predictive
     # validity testing to curtail some of the bad behavior of the model.
     thetas = data_interface.load_thetas(scenario_spec.theta)
+    if ((1 < thetas) | thetas < -1).any():
+        raise ValueError('Theta must be between -1 and 1.')
+    if (beta_params['sigma'] - thetas >= 1).any():
+        raise ValueError('Sigma - theta must be smaller than 1')
+
```

Comment:
I'm sure I've seen this check before. Any idea if that PR supersedes this or if it's duplicated logic?

Reply:
This just moves the check